modules/bitgo/src/v2/coins/utxo/recovery/crossChainRecovery.ts

```diff
@@ -95,7 +95,7 @@ async function getWallet(bitgo: BitGo, coin: AbstractUtxoCoin, walletId: string)
   try {
     return await coin.wallets().get({ id: walletId });
   } catch (e) {
-    if (e.status >= 400 && e.status < 500) {
+    if (e.status < 400 || e.status >= 500) {
```

Comment:
I wouldn't expect a status < 400 to be an error. Those should be successful responses from the api, yes?

Reply:
It should only enter the catch block if the response is an error though, no?

Reply:
Status' below 400 aren't error reponses.

Reply:
If we are seeing an error below 400 in the catch block, we're dealing with a logic error that needs to be fixed.

Reply:
oh right, yes I get what you mean now. So perhaps instead it should be:

```suggestion
    if (e.status >= 500) {
```

Reply:
After discussing with @mmcshinsky-bitgo in slack, the plan is to:
- go with `e.status >= 500`, 
- add a comment to explain it in the code,
- create a ticket for the backlog to better gracefully fail different errors for different wallet versions